roles/pre-ansimail/files/ansimail_scripts/helpers/verbose-fix

```diff
@@ -1,20 +1,20 @@
 #!/bin/sh
 
-if [ $(id -u) -ne 0 ]; then
+if [ "$(id -u)" -ne 0 ]; then
 	echo "This can only be run by root"
 	exit 1
 fi
 
-owner=$1
-group=$2
-mode=$3
-mfile=$4
+owner="$1"
+group="$2"
+mode="$3"
+mfile="$4"
 
-old_stat=$(stat -f '%u %g %OLp' $4)
-new_stat="$(id -u $1) $(getent group $2 | cut -d: -f3) $3"
```

Comment:
I noticed that you changed the variable names from 1,2,3,4 to the proper ones!  
Thanks a lot for that.  
 Could you also change these ones? It is better to do them all together.

Reply:
Sure! Just missed this spot.